shell/models/fleet.cattle.io.cluster.js

```diff
@@ -79,6 +79,14 @@ export default class FleetCluster extends SteveModel {
     return false;
   }
 
+  get canChangeWorkspace() {
+    return !this.isRke2 && !this.isLocal;
+  }
+
+  get isLocal() {
+    return this.metadata.name === 'local' || this.metadata?.labels?.[FLEET_LABELS.CLUSTER_NAME] === 'local';
```

Comment:
I thought we had somewhere the `'local'` as static configuration but we don't. Feel free to add or resolve this comment without changes.